scalastyle-config.xml

```diff
@@ -10,7 +10,7 @@
  <check level="warning" class="org.scalastyle.scalariform.SpacesAfterPlusChecker" enabled="true"/>
  <check level="warning" class="org.scalastyle.file.WhitespaceEndOfLineChecker" enabled="true"/>
  <check level="warning" class="org.scalastyle.scalariform.SpacesBeforePlusChecker" enabled="true"/>
- <check level="warning" class="org.scalastyle.file.FileLineLengthChecker" enabled="true">
+ <check level="warning" class="org.scalastyle.file.FileLineLengthChecker" enabled="false">
   <parameters>
```

Comment:
Having FileLineLengthChecker disabled, parameters can be taken off. 
e.g. `<check level="warning" class="org.scalastyle.file.FileLineLengthChecker" enabled="false" />`

Reply:
Removed